### PR TITLE
Update ap vendor packages

### PR DESCRIPTION
**Pull request by himabindu07**
## Description

Image Name |  Old-tag|  New-tag 
-- | -- | -- 
ap-nginx-es | 1.25.5 |1.27.0
ap-grafana | 10.4.5 |10.4.6
ap-postgresql | 15.6.0-1 | 15.7.0-1
ap-node-exporter | 1.8.1-1 | 1.8.2

## Related Issues
https://github.com/astronomer/issues/issues/6305

## Testing
QA should see all the components up and running 

## Merging

this PR should be cherry-picked into release-0.34, 0.35, 0.36

### CODE REVIEW

File: charts/postgresql/values.yaml [hunk 1]
```diff
@@ -15,7 +15,7 @@ global:
 image:
   registry: quay.io
   repository: astronomer/ap-postgresql
-  tag: 15.6.0-1
+  tag: 15.7.0-1
```
Comment (danielhoherd) [inline charts/postgresql/values.yaml:18]: There is currently no 15.7.0-1, only 15.7.0. https://quay.io/repository/astronomer/ap-postgresql?tab=tags

Comment (himabindu07) [reply charts/postgresql/values.yaml:18]: https://github.com/astronomer/ap-vendor/blob/main/postgresql/version.txt if you can see here its 15.7.0-1 only

Comment (himabindu07) [reply charts/postgresql/values.yaml:18]: Can I update this change ??

Comment (himabindu07) [reply charts/postgresql/values.yaml:18]: Updated the requested change

**Comment by pgvishnuram** [comment]
@himabindu07 can you remove nginx from this list

File: charts/postgresql/values.yaml (hunk 1 above)
Comment (pgvishnuram) [reply charts/postgresql/values.yaml:18]: I know the reason why it has not published

Comment (pgvishnuram) [reply charts/postgresql/values.yaml:18]: fixing it

Comment (pgvishnuram) [reply charts/postgresql/values.yaml:18]: this is fixed now

**Pull request merged**